### PR TITLE
V1 code review

### PR DESCRIPTION
**Pull request by imposyndro**

### CODE REVIEW

File: afile.txt [hunk 1]
```diff
@@ -10,4 +12,5 @@ developer 2 new line 5
 line 6
 line 7
 line 8
-This file was modified by developer 2
\ No newline at end of file
+
+This file was modified by developer 1 and 2
```
Comment (imposyndro) [inline afile.txt:15]: Changes are good

File: afile.txt [hunk 2]
```diff
@@ -10,4 +12,5 @@ developer 2 new line 5
 line 6
```
Comment (imposyndro) [inline afile.txt:12]: Wow

**Review by imposyndro — COMMENTED** [review]
Changes are good

**Comment by dwisneski** [comment]
you were supposed to leave the pull request in open state so I could comment on it.

**Pull request closed without merging**